app/lib/config.sh

```diff
@@ -7,7 +7,7 @@ set -euf
 
 config_get() {
 	path="$(config_path "$1")"
-	[ ! -r "$path" ] || cat "$path"
+	[ ! -r "$path" ] || envsubst "`printf '${%s} ' $(sh -c "env|cut -d'=' -f1")`" < "$path"
```

Comment:
What exactly am I trying to do? I thought it was expanding envvars inside of files in config/environments/

Reply:
that is correct, however this causes several additional processes to be spawned and run across all of dab every time any config value is retrieved. The change required to implement this should then affect the least used function as much as possible, in this the effect can be achieved in the `config_load_envs` function where it would not affect every other config operation.